src/app/view/timestamp-editor/timestamp-editor.component.ts

```diff
@@ -126,7 +130,7 @@ export class TimestampEditorComponent implements OnInit {
     this.selectablePublisherRoles$ = this.selectedTimestampDefinition$.pipe(
       map(timestampDefinition => this.timestampMappingService.overlappingPublisherRoles(timestampDefinition)),
       mergeMap(publisherRoles => {
-        return this.publisherRoleService.getPublisherRoleDetails().pipe(
+        return this.allPublisherRoles$.pipe(
```

Comment:
We do not return publisher roles for accept/rejection cases. This stems from `this.selectedTimestampDefinition$` not being defined for these cases which was overseen. This leads to location requirement, publisher and all else dependent not being set correctly

Reply:
Thanks. Fixed.

Reply:
It is fixed for this specific issue, but location requirements is also dependent on a NULL selectedTimestampDefintion - this might be out of scope for this issue.

Reply:
That logic should be triggered now due to that call to `updateTimestampDefinition()` (which in turn calls `this.updateLocationNameRequirement(timestampDefinitionTO);`).

Can you come with a concrete example where this does not work?